# Написать функцию my_add (), которая принимает два аргумента.
# Если аргументы можно сложить, возвращается сумма.
# Если сложение аргументов вызовет ошибку, функция должна вместо этого вернуть None.
# Если есть различные исключения - то их обрабатывать отдельно

def my_add(x, y):
    try:
        z = x + y
        print('Сумма двух чисел равна:', z)
    except Exception:
        print(None)


my_add(6, 12)
my_add(666, "-")
